content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -129,7 +129,7 @@ operator.
 
 
 * Read the {{< glossary_tooltip text="CNCF" term_id="cncf" >}}
-  [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
+  [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/main/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```

Comment:
Linking to a branch can lead to a broken link
```suggestion
  [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/163962c4b1cd70d085107fc579e3e04c2e14d59c/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```

Reply:
Got it. Updated. Thanks. @sftim